doc/dev/release.rst

```diff
@@ -216,7 +214,7 @@ so:
         virtualenv build
         cd build
         source bin/activate
-        pip install -U setuptools==3.4.1 wheel
+        pip install wheel
```

Comment:
@betatim recent setuptools is great! The reason to test with older is to determine the true minimum version needed so that can be communicated to users (or caught at runtime). Unless you want the install guide to start with `pip install -U setuptools` :-)